code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2172,3 +2182,36 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjust_fire_stacks(1)
 		M.IgniteMob()
 	..()
+
+/datum/reagent/consumable/ethanol/painkiller
+	name = "Painkiller"
+	description = "Dulls your pain. Not sure if that's a good thing or a bad thing."
+	boozepwr = 20
+	color = "#EAD677"
+	quality = DRINK_NICE
+	taste_description = "sugary tartness"
+	glass_icon_state = "painkiller"
+	glass_name = "Painkiller"
+	glass_desc = "A combination of tropical juices and rum. Surely this will make you feel better."
+
+/datum/reagent/consumable/ethanol/painkiller/on_mob_life(mob/living/carbon/M)
+	// Causes the health doll to display blue regardless of damage, until the drink is metabolized.
+	M.hal_screwyhud = SCREWYHUD_HEALTHY
+	..()
+
+/datum/reagent/consumable/ethanol/painkiller/on_mob_end_metabolize(mob/living/M)
+	if(iscarbon(M))
+		var/mob/living/carbon/N = M
+		N.hal_screwyhud = SCREWYHUD_NONE
```

Comment:
is this how other sources do it? There's no source appended so if you're taking say miner salve and painkiller, it will remove the hud even though it should still have it from miner salve.

Reply:
this will remove the hud then next tick miners salve will restore it because they both use it in on mob life

Reply:
To answer your question, this is how the one other source does it. However: Given that accounting for interaction will be more complicated than anticipated, and I’m not particularly married to the idea, I’m going to go ahead and remove the effect for now in the interest of the PR as a whole. I will say I think it was a good idea and I’d like to work on adding it back in later. But adding in pineapple juice is more important, so expect an update reverting this change.

Reply:
@JJRcop true, but that means it will potentially show you the true HP which will ruin the point of the concoctions.